### ISSUE TITLE
Can't run example on MAC with metal backend

### ISSUE THREAD

**Issue by trsh** [amethyst/rendy#262]
`cargo run --features metal`

gives

```thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: No enabled backends among:  vulkan  dx12  metal  gl', src/libcore/result.rs:1165:5```

On windows machine `cargo run --features vulkan` works

[example.zip](https://github.com/amethyst/rendy/files/4083492/example.zip)

**Comment by omni-viral** [comment]
Oh, I thought we eliminated all bugs with metal backend activation.
Could be rustfmt messing with complex `cfg` macro again. I'll check it

**Comment by omni-viral** [comment]
Oh, I thought we eliminated all bugs with metal backend activation.
Could be rustfmt messing with complex `cfg` macro again. I'll check it

**Comment by trsh** [comment]
Thank you!

**Comment by StarArawn** [comment]
Is there any progress on this? I also get the same error.

**Comment by trsh** [comment]
There seems to be zero progress on bug handling in this project :(

**Comment by termhn** [comment]
Yes, the primary maintainer(s) have been busy with other things for some
time now.

**Comment by jmitchell89** [comment]
I would try to fix, but taking a look at it the macro's are hard for me to follow and a bit over my head at the moment.

**Comment by StarArawn** [comment]
I would try to fix, but taking a look at it the macro's are hard for me to follow and a bit over my head at the moment. 

I have debugged this a bit further and metal shows up as an available adapter for me it just doesn't get enabled.
